skimage/_vendored/numpy_lookfor.py

```diff
@@ -132,7 +132,9 @@ def _lookfor_generate_cache(module, import_modules, regenerate):
             for n, v in _getmembers(item):
                 try:
                     item_name = getattr(
-                        v, '__name__', "%s.%s" % (name, n)  # noqa: UP031
+                        v,
+                        '__name__',
+                        "%s.%s" % (name, n),  # noqa: UP031
```

Comment:
Wonder why we have `noqa` here?

Reply:
I was planning to do a search for `noqa` once he release is out. I will make a follow-up PR incorporating your comments.